x/programs/rust/wasmlanche-sdk/src/memory.rs

```diff
@@ -54,9 +48,7 @@ pub fn into_bytes(ptr: *const u8) -> Option<Vec<u8>> {
 /// Panics if the pointer exceeds the maximum size of an isize or that the allocated memory is null.
 #[no_mangle]
 pub extern "C" fn alloc(len: usize) -> *mut u8 {
-    if len == 0 {
-        return std::ptr::null_mut();
-    }
+    assert!(len > 0, "cannot allocate 0 sized data");
```

Comment:
Much nicer!

Reply:
😂  I just put the file back to what it was (I think you wrote that code)